tests/unit/conftest.py

```diff
@@ -0,0 +1,11 @@
+from typing import Dict
+
+import pytest
+
+
+@pytest.fixture()
+def noise_requirements() -> Dict:
```

Comment:
What are you envisioning this being used for?

Reply:
I removed it. It was lingering from a test I decided against after chatting w/ Matt